src/docs/product/integrations/notification-incidents/msteams/index.mdx

```diff
@@ -41,6 +41,12 @@ Use Microsoft Teams for [alerts](/product/alerts/) regarding issues, environment
 
 ### Alert Rules
 
+<Note>
+
+If Microsoft Teams is not appearing as an option in issue alert actions, re-install the integration using "add to a team".
```

Comment:
```suggestion
If Microsoft Teams is not appearing as an option in issue alert actions, re-install the integration using "Add to a team".
```

Reply:
I also noticed it says "Add to team" instead of "Add to a team" in lines 23 & 28. It should be "Add to a team" to be consistent with what the UI says (assuming the screenshot is correct).